src/components/dropdownSearch/DropdownSearchConnected.tsx

```diff
@@ -11,26 +11,29 @@ import {
   toggleDropdownSearch,
   updateActiveOptionDropdownSearch,
 } from './DropdownSearchActions';
-import { defaultSelectedOption, IDropdownSearchState } from './DropdownSearchReducers';
+import { IDropdownSearchState } from './DropdownSearchReducers';
 
 const mapStateToProps = (state: IReactVaporState, ownProps: IDropdownSearchProps): IDropdownSearchStateProps => {
   const dropdownSearch: IDropdownSearchState = _.findWhere(state.dropdownSearch, { id: ownProps.id });
-  const selectedOption = ownProps.defaultSelectedOption || defaultSelectedOption;
 
   if (dropdownSearch) {
     return {
-      isOpened: dropdownSearch.isOpened,
-      options: dropdownSearch.options,
-      selectedOption: dropdownSearch.selectedOption || selectedOption,
+      isOpened: dropdownSearch.isOpened || false,
```

Comment:
If you are able to move a few of these to the default props, that would make it cleaner here. Example: if you set `isOpened: false` as a default prop, you can simply write `isOpened: dropdownSearch.isOpened` here and whomever uses the component without Redux won't have to pass it all Redux props.